rs/nns/governance/unreleased_changelog.md

```diff
@@ -4,14 +4,15 @@ In general, upcoming/unreleased behavior changes are described here. For details
 on the process that this file is part of, see
 `rs/nervous_system/changelog_process.md`.
 
-
 # Next Upgrade Proposal
 
 ## Added
 
 ## Changed
 
 * Increased the probability of failure from 70% to 90% for the deprecated _pb methods.
+* Disable Neuron's Funds for ongoing SNSs, as approved in
+  proposal [135970](https://dashboard.internetcomputer.org/proposal/135970).
```

Comment:
Note: DO NOT MERGE unless community passes this proposal.

Reply:
You may know this, but FYI...

GitHub will block merge if you put "DO NOT MERGE" at the beginning of PR title, because PR titles MUST use a Conventional Commits prefix. This is what I do.

It used to be that merge would get blocked if `DO NOT MERGE` appears in any files, but I think they took that away (why???). This might have happened when we switched to GitHub.

Reply:
> It used to be that merge would get blocked if DO NOT MERGE appears in any files
That still works.

I am aware of both features, but wanted to just remind myself and any other reviewers.